sig-autoscaling/annual-report-2021.md

```diff
@@ -0,0 +1,69 @@
+# SIG Autoscaling Annual Report 2021
+
+## Operational
+
+* How are you doing with operational tasks in
+[sig-governance.md](https://git.k8s.io/community/committee-steering/governance/sig-governance.md)?
+  * Is your README accurate? have a CONTRIBUTING.md file?
+    * Yes, [README.md](https://github.com/kubernetes/community/blob/master/sig-autoscaling/README.md) is accurate (other than one link)
+    * We do not currently have a [CONTRIBUTING.md] in the community repo
+
+  * All subprojects correctly mapped and listed in [sigs.yaml](https://git.k8s.io/community/sig-list.md)?
+    * [Yes](https://github.com/kubernetes/community/tree/master/sig-autoscaling#subprojects)]
+
+  * What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping
+  recordings up to date/trends in community members watching recordings?
+    * The main SIG meeting has ~8 people on average. Meeting notes [are recorded](https://docs.google.com/document/d/1RvhQAEIrVLHbyNnuaT99-6u9ZUMp7BfkPupT2LAZK7w/edit#heading=h.mh9yomoq1p6v), no video recordings made.
```

Comment:
Why aren't recordings being made?

Reply:
A good question. 

I don't think we've made recordings of the meetings for as long as I've been involved in the SIG. I've added discussing beginning to make recordings and make them available to the agenda for the next SIG office hours. I'm assuming the [guidelines here](https://github.com/kubernetes/community/blob/master/communication/youtube/youtube-guidelines.md) are the go-to reference for setting this up?

Reply:
Yup, that's the right one!

Also for reference, providing publicly recorded meetings is a [requirement](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#sig-roles-and-organizational-governance) of all sigs. If you have questions or concerns about doing this in principle, please come talk to Steering!

If it's just a matter of getting things set up from a technical standpoint to enable this, come talk to contribex :)